From 34fd55fe55e9d13f9c3f114d6fb512cc41442373 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Tue, 6 Apr 2021 15:44:12 -0500
Subject: [PATCH 118/120] HSD #22012814951-1: intel_fcs: don't report error
 with rejected service

commit 61aa19580630f72cbd4093359da786dbb1886247 from
https://github.com/altera-opensource/linux-socfpga.git

Change to not report mailbox error when service request was rejected.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/crypto/intel_fcs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/intel_fcs.c b/drivers/crypto/intel_fcs.c
index b13746df2ff5..f203113c13a5 100644
--- a/drivers/crypto/intel_fcs.c
+++ b/drivers/crypto/intel_fcs.c
@@ -68,7 +68,7 @@ static void fcs_data_callback(struct stratix10_svc_client *client,
 		priv->size = (data->kaddr3) ?
 			*((unsigned int *)data->kaddr3) : 0;
 	} else {
-		dev_err(client->dev, "rejected\n");
+		dev_warn(client->dev, "rejected, invalid param\n");
 		priv->status = -EINVAL;
 		priv->kbuf = NULL;
 		priv->size = 0;
-- 
2.31.1

